config/src/config_wrapper.cpp

```diff
@@ -72,21 +56,146 @@ void load()
     dynamic_config.load_from_file("dynamic_config.json");
 
     loaded = true;
+    log << "Loaded files";
+}
+
+int32_t number_of_stations()
+{
+    load();
+    try {
+        static auto tmp = static_config.getSize("stations") + static_config.getSize("end_stations");
```

Comment:
Helt generelt: Jeg er ikke glad ved navngivningen `tmp`, der er IMO ingen skade i at gentage hvad der ligger i værdien frem for at man skal kigge X linjer op for at finde det, især når udtrykket som her ikke er trivielt.